libcudacxx/include/cuda/std/__cccl/dialect.h

```diff
@@ -90,4 +90,25 @@
 #  define _CCCL_TRAIT(__TRAIT, ...) __TRAIT<__VA_ARGS__>::value
 #endif // _CCCL_STD_VER <= 2014
 
+// In nvcc prior to 11.3 global variables could not be marked constexpr
+#if defined(_CCCL_CUDACC_BELOW_11_3)
+#  define _CCCL_CONSTEXPR_GLOBAL const
+#else // ^^^ _CCCL_CUDACC_BELOW_11_3 ^^^ / vvv !_CCCL_CUDACC_BELOW_11_3 vvv
+#  define _CCCL_CONSTEXPR_GLOBAL constexpr
+#endif // !_CCCL_CUDACC_BELOW_11_3
+
+// Inline variables are only available from C++17 onwards
+#if _CCCL_STD_VER >= 2017 && defined(__cpp_inline_variables) && (__cpp_inline_variables >= 201606L)
+#  define _CCCL_INLINE_VAR inline
+#else // ^^^ C++14 ^^^ / vvv C++17 vvv
+#  define _CCCL_INLINE_VAR
+#endif // _CCCL_STD_VER <= 2014
```

Comment:
Question: wouldn't this suffice:
```suggestion
#ifdef __cpp_inline_variables
#  define _CCCL_INLINE_VAR inline
#else
#  define _CCCL_INLINE_VAR
#endif
```
I know feature testing macros are officially part of C++20, but available a lot earler in many compilers. Also, there should be no need to test the feature version or the C++ version, just whether the compiler understands inline variables.

Reply:
I believe older gcc compilers do not properly support inline variables despite defining the feature test macros

Reply:
Alright. Consider adding a comment on this :)